README.md

```diff
@@ -218,6 +223,50 @@ To start with the actual implementation, you may check our [IntelliJ Platform SD
 For those, who value example codes the most, there are also available [IntelliJ SDK Code Samples][gh:code-samples] and [IntelliJ Platform Explorer][jb:ipe] – a search tool for browsing Extension Points inside existing implementations of open-source IntelliJ Platform plugins.
 
 
+## Testing
+
+[Testing plugins][docs:testing-plugins] is an essential part of the plugin development for the IntelliJ-based IDEs to make sure that everything works as expected between IDE releases and plugin refactorings.
+The IntelliJ Platform Plugin Template project provides examples of two testing approaches – functional and UI tests.
+
+### Functional tests
+
+Most of the IntelliJ Platform codebase tests are model-level, run in a headless environment using an actual IDE instance.
+The tests usually test a feature as a whole rather than individual functions that comprise its implementation, like in unit tests.
+
+Depending on the feature type and its execution time, there are two types of tests available in the IntelliJ Platform test framework:
```

Comment:
I'd drop that paragraph until `In src/test/...` and just link to https://plugins.jetbrains.com/docs/intellij/testing-plugins.html
which has all up to date information

Reply:
Ok, that link is already available above, so I'll just remove it.